plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -226,8 +228,8 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
    cfg.add_options()
          ("chain-alias", bpo::value<string>(), "Chain Alias, alias id mapping is needed. a bootnode server address must set in p2p-peer-address.")
          ("chain-id", bpo::value<string>(), "Chain ID, must match chain id in genesis or snapshot or block. a bootnode server address must set in p2p-peer-address.")
-         ("chain-alias-mapping", bpo::value< vector<string> >()->composing(),
-          "The mapping info in config.\n"
+         ("chain-alias-mapping", bpo::value< vector<string> >()->composing()->multitoken(),
```

Comment:
The alias to chain_id mappings should not be a configuration option, but rather should be part of the code distribution. For instance, it could be loaded as a .json file upon application start, that would exist in a specific folder.

The objective is to have an alias supported by default in the code base for the coalition chains, without requiring node operators to define these mappings themselves.

As new chains join the coalition, we can leverage the PR mechanism to have them added to the list of aliases.

Here are the mappings :

"eos" : "aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906"
"wax": "1064487b3cd1a897ce03ae5b6a865651747e2e152090f99c1d19d44e01aea5a4"
"telos" : "4667b205c6838ef70ff7988f6e8257e8be0e1284a2f59699054a018f743b1d11"
"ux" : "8fc6dce7942189f842170de953932b1f66693ad3788f766e777b6f9d22335c02"